xcell/cls/cl.py

```diff
@@ -244,6 +244,20 @@ def get_cl_file(self):
                 cl_cov_12_cp *= correction
                 cl_cov_22_cp *= correction
 
+            if (mapper1.wf is not None) or (mapper2.wf is not None):
```

Comment:
What is wf?

Reply:
window function. It is meant to represent any cl level correction in the spirit of Lenz's wl_eff

Reply:
Removed and moved to mapper_base
https://github.com/xC-ell/xCell/blob/830b3f246ce96fb57de03f4b5ec9c6831da52cd9/xcell/mappers/mapper_base.py#L56-L65